tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -189,6 +181,12 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
     assertCpuGpuMatchesRegexpFind(patterns, Seq("", "\u0007", "a\u0007b", 
         "\u0007\u003f\u007f", "\u007f", "\u007f2"))
   }
+
+  test("hex digits >= 0x7F - find") {
```

Comment:
Also you should merge this with the other hex digit to test all of them in the same unit test.

Reply:
I agree. The scala test can be removed in favor of the python test provided that test covers all cases